client/app/components/sections/Topbar/Topbar.js

```diff
@@ -197,12 +173,43 @@ class Topbar extends Component {
       }) :
       null;
 
+    // menu props
+    const hasMenuContext = !!this.props.menu;
+    const menuLinksData = hasMenuContext ?
+      this.props.menu.links.map((l) => (
+        {
+          active: l.link === location,
+          activeColor: marketplaceColor1,
+          content: l.title,
+          href: l.link,
+          type: 'menuitem',
+          priority: l.priority,
+          external: l.external,
+        }
+      )) :
+      [];
+    const menuProps = hasMenuContext ?
+      Object.assign({}, this.props.menu, {
+        key: 'menu',
+        name: t('web.topbar.more'),
+        nameFallback: t('web.topbar.menu'),
+        identifier: 'Menu',
+        limitPriorityLinks: this.props.menu.limit_priority_links,
+        menuLabelType: LABEL_TYPE_DROPDOWN,
+        menuLabelTypeFallback: LABEL_TYPE_MENU,
+        content: menuLinksData,
+      }) :
+      {};
 
-    const pathParams = { return_to: location };
-    const loginRoute = this.props.routes.login_path ? this.props.routes.login_path(pathParams) : '#';
-    const signupRoute = this.props.routes.sign_up_path ? this.props.routes.sign_up_path() : '#';
-
-    const mobileMenuProps = this.props.menu ?
+    // mobile menu props
+    const isAdmin = !!(this.props.user && this.props.user.isAdmin && loggedInUsername);
```

Comment:
`isAdmin` is also used in `ProfileDropdown`, so maybe not under "mobile menu props"